src/Providers/HookServiceProvider.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Carnival\Providers;
+
+use Illuminate\Support\ServiceProvider;
+use Carnival\Hooks\HookManager;
+
+class HookServiceProvider extends ServiceProvider
+{
+    /**
+     * Register services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        /**
+         * Bind Hook Service
+         */
+        $this->app->bind('hooks', fn() => new HookManager);
+    }
+
+    /**
+     * Bootstrap services.
+     *
+     * @return void
+     */
+    public function boot()
```

Comment:
I *think* that we don't need to have a boot method. I don't have one in the `ExtensionsServiceProvider`

Of course maybe there'll be boot logic here one day, but we could re-add the method if that day comes

Reply:
Sounds good